.github/workflows/shellcheck_reviewdog.yml

```diff
@@ -0,0 +1,19 @@
+name: shellcheck-reviewdog
+on: [pull_request]
+jobs:
+  shellcheck:
+    name: runner / shellcheck
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: shellcheck
+        uses: reviewdog/action-shellcheck@v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          reporter: github-pr-review
+          level: info
+          filter_mode: added # This determines the scope of what reviewdog will report
+          fail_on_error: true
+          path: "."
+          pattern: "*.sh"
```

Comment:
I have a problem with this line, the way it is the `tests/utils` file is being ignored. From the [action documenation](https://github.com/reviewdog/action-shellcheck#pattern):

> ### `pattern`
> Optional. File patterns of target files. Same as -name [pattern] of find command. Default: *.sh

Maybe I just don't understand `find` enough, but I can't seem to create multiple patterns. If anyone could help me with this I would appreciate it. A somewhat dirty workaround would be to create another reviewdog step just for utils, but this seems a bit "gambiarra" to me

Reply:
we could also just rename utils to utils.sh haha it always bugged me that it is the only one without a file extension

Reply:
Apparently `find` takes in a glob, and I think you can match multiple patterns like this:
```
{utils, *.sh}
```

Reply:
@kwy95, [here](https://github.com/alan-barzilay/kworkflow/pull/1/checks?check_run_id=2810073860) I tried using the line `           pattern: "{utils,*.sh}" `, but it didn't work. Also, I tried using that with find on the command line and couldn't make it work haha  how exactly should be the find command?

Reply:
Is there a way to activate extended globbing in the test? It's usually done with `shopt -s extglob`

Reply:
With `extglob` you can try this:
```
@(utils|*.sh)
```

Reply:
@kwy95 not sure, I never used glob that much so I'm kinda lost. Btw, i found [where this is evaluated](https://github.com/reviewdog/action-shellcheck/blob/master/entrypoint.sh):
 `$(find "${INPUT_PATH:-'.'}" -not -path "${INPUT_EXCLUDE}" -type f -name "${INPUT_PATTERN:-'*.sh'}")`
what we are providing is that `INPUT_PATTERN` variable

Reply:
`-name "utils" -or -name "*.sh"` works perfectly, but I don't think we can escape from the variable in a way to input more flags

Reply:
I'm a little bit late to the discussion, but I also agree that we can add `.sh` to the end of `utils`.

Reply:
I'm going to merge your change first, and later we can update the `utils` file.